### PR TITLE
Change node vers.

### PR DESCRIPTION
**Pull request by crandmck**
@smartmouse Per your request in Slack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ For more information, see the <a href="http://loopback.io/doc/en/lb3/Oracle-conn
 
 ## Prerequisites
 
-**Node.js**: The Oracle connector is compatible with Node v.0.10.x, v.12.x, and later.  
+**Node.js**: The Oracle connector requires with version 4.x or 6.x.
```
Comment (cjbj) [inline README.md]: Typo 'requires with'

**Comment by crandmck** [comment]
@cjbj tyvm.  Fixed!
Sloppy on my part. :-(

**Comment by loay** [comment]
@slnode test please

**Comment by smartmouse** [comment]
LGTM

**Pull request merged**